Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -278,7 +278,7 @@ view push state =
                     [ width MATCH_PARENT
                     , height MATCH_PARENT
                     , background Color.transparent
-                    , padding (PaddingBottom if showLabel then (if os == "IOS" then 46 else 70) else (if os == "IOS" then 10 else 34))
+                    , padding (PaddingBottom if showLabel then (if os == "IOS" then 53 else 70) else (if os == "IOS" then 10 else 34))
```

Comment:
can you check this padding for IOS with @Shikhar-24, with this changed value it might not work as expected when native changes are merged for IOS

Reply:
this is tested with native changes as well.